### PR TITLE
copy the code from trials project

### PR DESCRIPTION
**Pull request by AbelG101**

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
Good job so far!
There are some issues that you still need to work on! 💪 💪 💪

## Highlights
- Use of correct HTML semantics
- Use of correct GitHub flow

## Required Changes ♻️

- please make sure you've added these [linters ](https://github.com/microverseinc/linters-config) provided by Microverse and make sure they are set up correctly and they all pass with green checkmarks.
- please add a .gitignore file to the project.
- please make your commits more descriptive
- please use this [readme template](https://github.com/microverseinc/readme-template) and please be sure to edit all of the sections to better suit your project.
- please add a more descriptive pull request.
- please add styling to your style.css file.
_Please check the comments under the review as well._

## Optional suggestions 

- Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @AbelG101 in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @AbelG101 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly be noticed that you should write at least 2 comments in the review.
- [ ] Also you should use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments and make sure to leave the comment under a line or block of code that fits the context the comment is about.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (AbelG101) [inline index.html:7]: I think its better to use a more descriptive title than just Document

**Review by cynthiainga — APPROVED** [review]
Hi @AbelG101, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] Good tone with the student :heavy_check_mark:

## Optional suggestions

- [ ] You did a good job, I think it should be better if you used more a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**